integration_test/third_party_apps_data/applications/iis/metadata.yaml

```diff
@@ -15,6 +15,7 @@ minimum_supported_agent_version:
   metrics: 2.10.0
   logging: 2.10.0
 expected_metrics:
```

Comment:
Can we add the configuration option for the version flag in the section below?

Reply:
Added